chia/rpc/full_node_rpc_api.py

```diff
@@ -835,6 +838,38 @@ async def get_mempool_items_by_coin_name(self, request: dict[str, Any]) -> Endpo
 
         return {"mempool_items": [item.to_json_dict() for item in items]}
 
+    async def create_block_bundle_from_mempool(self, _: dict[str, Any]) -> EndpointResult:
+        mempool_bundle = None
+
+        # Grab best transactions from Mempool for given tip target
+        async with self.service.blockchain.priority_mutex.acquire(priority=BlockchainMutexPriority.high):
+            peak: Optional[BlockRecord] = self.service.blockchain.get_peak()
+
+            if peak is None:
+                return {"mempool_bundle": None}
+
+            # Finds the last transaction block before this one
+            curr_l_tb: BlockRecord = peak
+            while not curr_l_tb.is_transaction_block:
+                curr_l_tb = self.service.blockchain.block_record(curr_l_tb.prev_hash)
+
+            self.service.log.info("Beginning simulated block construction from mempool")
+            start_time = time.time()
+
+            try:
+                result = await self.service.mempool_manager.create_bundle_from_mempool(
```

Comment:
I think it would be better to call `create_block_generator()`, to also include the time it takes to serialize/compress the block